py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/config_base.py

```diff
@@ -34,4 +52,4 @@ def config_type(self) -> ConfigType: ...
     def export_schema(self) -> pyarrow.schema: ...
 
     @abstractmethod
-    def record_from_entity(self, entity: dict) -> dict: ...
\ No newline at end of file
+    def record_from_entity(self, entity: dict) -> dict: ...
```

Comment:
we should decide if we want this newline at the end of our files or not.

i think our editors are different and its causing this diff to appear.